File: IntroPC/Tarea 6/Rutina_perfectos/callperf.cpp
#include <cstdio>
#include <cmath>
#include "Fperfecto.cpp"
using namespace std;

int main (void)
{
  int i,perf,num,sw;
  printf("ingrese el numero limite para caulcular numeros perfectos: ");
  scanf("%d",&num);
  for (i=num;i>=1;i--)
    {
      perf=Fperfecto(i);
      if (perf == 1)
	{
	  sw=1;
	  printf("numero perfecto inferior a %d: %d \n",num,i);
	}
    }
  if (sw==0)
    {
      printf("no existen numeros perfectos inferiores al numero %d \n",num);
    }
}

File: IntroPC/Tarea 6/Rutina_perfectos/callperf.py
# /usr/bin/env python

import Fperfecto

num=input("ingrese el numero limite para caulcular numeros perfectos: ")
sw=0
i=num
while i>=1:
    perf=Fperfecto.Fperfecto(i)
   
    if perf == 1:
        sw=1
        print"numero perfecto inferior a %d: %d \n"%(num,i)
          
    i=i-1
if sw==0:
    print"no existen numeros perfectos inferiores al numero %d \n"%(num)
    
